form_app/pubspec.yaml

```diff
@@ -12,14 +12,12 @@ dependencies:
   cupertino_icons: ^1.0.0
   intl: any
   http: ^0.12.0
-  mockito: ^4.1.1
-  json_annotation: ^3.0.0
```

Comment:
> Stop pinning `json_annotation` and `json_serializable` since `flutter_test` also has version constraints for these, and it was hard to get a range that worked for both master and stable.

@RedBrogdon Is this a reasonable thing to do?  Or do you have a better suggestion?

You can see the problem if you check out flutter on master and then flutter on stable.  They want different `json_` version ranges.

Reply:
This seems like normal maintenance to me, is there something about `flutter_test` that makes this difficult to maintain?

Reply:
Yes, check out https://github.com/flutter/flutter/issues/82280.